### ISSUE TITLE
browser form validation is not output by JAWS

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#621]
### Summary

1. Go to https://codepen.io/jaws-test/pen/mdpgyPx
2. Press submit
3. Tab to all form fields

### Expected result

Each field has a tooltip with a detailed error description. This should be output by JAWS.

### Actual result

JAWS only outputs the status invalid, but not the error description or the browser's input instructions.

Only the message at the field which automatically receives the focus after submit is output by JAWS. The messages for the other fields are not perceptible with JAWS. Even if the intially focused field receives the focus again, the message is not perceptible. Thus, browser error messages cannot be used in conjunction with JAWS.

See: https://github.com/w3c/aria/issues/1488#issuecomment-1105479910 and https://github.com/FreedomScientific/VFO-standards-support/issues/14

Related: https://github.com/w3c/wcag/issues/961

#### JAWS version and build number
JAWS 2022
 
#### Operating System and version
Windows 10

#### Browser and version:
current version of Chrome, Edge, Firefox

**Comment by stevefaulkner** [comment]
>Each field has a tooltip with a detailed error description. This should be output by JAWS.

Its not announced by NVDA either. It appears that the 'tooltip' is not exposed as acc description (as i would expect).

**Comment by JAWS-test** [comment]
Yes, it is correct that NVDA does not output that either. The error message should not be output as a description but analogous to aria-errormessage. This is important because the form field can also have a description. Both must be perceptible: Description and error message.

**Comment by stevefaulkner** [comment]
@JAWS-test refer to https://github.com/w3c/aria/issues/1488#issuecomment-1106192927 which details why it is not an error message

**Comment by JAWS-test** [comment]
@stevefaulkner  See: https://github.com/w3c/aria/issues/1488#issuecomment-1106226394
